src/olympia/addons/tests/test_views.py

```diff
@@ -1731,7 +1731,9 @@ def _generate_source_zip(
         source.seek(0)
         return source
 
-    def test_source_zip(self):
+    @mock.patch('olympia.addons.views.log')
```

Comment:
what about statsd logging? Should we test for that too?

Reply:
When I was writing the patch to log and add timings to source code uploads via the views, I asked about writing tests for the `statsd` timings, as I wasn't sure how to do it, and I believe we decided it was unnecessary, so I didn't do it here either.